src/components/AddPoll.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div id="addpoll">
+    <section>
+      <div class="field">
+        <b-field label="Title">
+          <input class="input" name="title" type="text" placeholder="Title input" v-model="title">
+        </b-field>
+      </div>
+      <table class="table">
+        <div>
+        <button class="button is-danger" @click="addRow">Add row</button>
+      </div>
+        <tbody>
+          <tr v-for="(item,index) in getRows" :key="index">  
+            <td>  
+              <b-field label="Option">
+                <input class="input" type="text" placeholder="Option" v-model="item.option">
+              </b-field>
+            </td>
+            <td>
+              <a v-on:click="removeElement(index);" style="cursor: pointer">Remove</a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <div class="field">
+          <button class="button is-primary" v-on:click="addpollSubmit" :class="{ 'button': true, 'is-full' : true, 'is-loading' : login_progress}">Register</button>
+        </div>
+    </section>
+  </div>
+</template>
+
+<script>
+import { mapActions, mapGetters } from "vuex";
+import { mapFields } from "vuex-map-fields";
+
+export default {
+  name: "AddPollForm",
+  watch: {
+    isSuccess: function(val) {
+      if (val === true) {
+        this.$router.push("profile");
+      }
+    },
+    error_message: function(val) {
+      if (val) {
+        this.$snackbar.open({
+          duration: 5000,
+          message: "Add Poll is Fail " + val.message,
+          type: "is-danger",
+          position: "is-botton-left"
+        });
+      }
+    }
+  },
+  computed: {
+    ...mapFields(["option1", "option2", "option3", "option4", "title"]),
```

Comment:
this should be dynamic?? already talked about this "option1", "option2", "option3", "option4